resources/resolver.go

```diff
@@ -377,9 +378,56 @@ func ResolveResources(
 			}
 		}
 	}
+
+	flagVocabExists := CheckFileDoesNotExist(path.Join(*dir, "vocab.json"))
+
+	if flagVocabExists {
```

Comment:
These flags are inverse of the usual case.  The variable name implies that the Boolean is testing for existence, and `true` means it does exist.  But it's actually `false` where it does exist.

It should be `flagVocabNotExist` or make the function call `CheckFileExist`. Either way, it should be `!flagVocabExists` for this and the following.